src/pages/api/orgs/[orgId]/events.ics/index.ts

```diff
@@ -0,0 +1,74 @@
+import dayjs, { Dayjs } from 'dayjs';
+import { NextApiRequest, NextApiResponse } from 'next';
+
+import BackendApiClient from 'core/api/client/BackendApiClient';
+import { ZetkinEvent } from 'utils/types/zetkin';
+
+const timeStamp = (t: Dayjs): string => t.format('YYYYMMDDTHHmmss');
+const formatString = (str: string): string =>
+  str
+    .replaceAll('\n', '\\n')
+    .match(/.{1,75}/g)
+    ?.join('\n\t') ?? '';
+
+export default async function handle(
+  req: NextApiRequest,
+  res: NextApiResponse
+): Promise<void> {
+  const { orgId } = req.query;
+
+  const apiClient = new BackendApiClient(req.headers);
+
+  const startTime = dayjs()
+    .subtract(2, 'month')
+    .toDate()
+    .toISOString()
+    .slice(0, 10);
+
+  const events = await apiClient.get<ZetkinEvent[]>(
+    `/api/orgs/${orgId}/actions?frecursive&ilter=start_time>=${startTime}`
+  );
+
+  const vEvents: string[] = [];
+  events
+    .filter((event) => !event.cancelled && event.published)
+    .forEach((event) => {
+      vEvents.push(`BEGIN:VEVENT`);
+      vEvents.push(`UID:${event.id}@zetkin.org`);
+      vEvents.push(
+        `ORGANIZER;CN=${event.organization.title.replace(
+          '\n',
+          ' '
+        )}:MAILTO:noreply@zetkin.org`
```

Comment:
Instead of this hardcoded email address, could we maybe load the organization details from `/api/orgs/<ID>` to get the organization data and use the email address of the organization, falling back to the no-reply? Ideally I'd like no email address at all for those cases where the organization does not have an address, but I understand your comment in the PR as saying that's not valid ICS.